packages/build/gulpfile.js

```diff
@@ -52,7 +52,9 @@ gulp.task('compile', () => {
       .pipe(gulp.dest('lib'));
 });
 
-gulp.task('test', ['build'], function() {
+gulp.task('test', ['build', 'test:unit']);
+
+gulp.task('test:unit', function() {
```

Comment:
Looks like build is missing a corresponding change to its package.json?

Reply:
Good catch.  Added